types/global.d.ts

```diff
@@ -230,7 +235,7 @@ declare global {
     upgrades: CustomHardhatUpgrades;
     network: Omit<Network, 'name'> & { name: keyof typeof networks };
     ethers: typeof ethers;
-    getSigners: () => Promise<(Signer & TypedDataSigner)[]>;
+    getSigners: () => Promise<Signer[]>;
```

Comment:
should this be `| FireBlocksSignerTypeHere` ?

Reply:
Even still, in case the two types deviate in the future we won't know if we don't make the differentiation here.

Reply:
also, `FireblocksSigner['_fireblocksWeb3Provider']` is only valid in one context afaict